jboss-tck-runner/pom.xml

```diff
@@ -21,8 +21,10 @@
     <url>http://weld.cdi-spec.org</url>
 
     <properties>
-        <htmlunit.version>2.9</htmlunit.version>
-        <shrinkwrap.version>1.0.1</shrinkwrap.version>
+        <!-- This matches the htmlunit version in TCK -->
+        <htmlunit.version>2.50.0</htmlunit.version>
+        <!-- Make the surefire plugin version explicit -->
+        <surefire.version>3.0.0-M5</surefire.version>
```

Comment:
The very same version is declared by weld-parent project that is a `<parent>` of this one. So you should be already getting this version.
See https://github.com/weld/parent/blob/44/pom.xml#L112